.circleci/config.yml

```diff
@@ -57,3 +57,4 @@ jobs:
             command: |
               ./b2 -j 2 cxxstd=latest libs/beast/test
               ./b2 -j 2 cxxstd=latest libs/beast/example
+            no_output_timeout: 20m
```

Comment:
Do these changes cause the tests to produce no output for 20 minutes?

Reply:
Difficult to say whether it was these changes or just variance at circleci (which I suspect). The default is 10 mins.